doc/examples.rst

```diff
@@ -12,6 +12,7 @@ with different frameworks:
 - `A PyQt4 application <https://github.com/takluyver/pynsist/tree/master/examples/pyqt>`_
 - `A PyGTK application <https://github.com/takluyver/pynsist/tree/master/examples/pygtk>`_
 - `A PyGTK application including Numpy and Matplotlib (32 bit, Python 2.7) <https://github.com/takluyver/pynsist/tree/master/examples/pygtk_mpl_numpy>`_
+- `A PyGTK application including Numpy and Matplotlib (64 bit, Python 3.4) <https://github.com/takluyver/pynsist/tree/master/examples/pygi_mpl_numpy>`_
```

Comment:
pygtk -> pygi

Truth be told, I'm never entirely sure what to call this. It officially seems to be PyGObject, but there's a package with the same name using the old direct bindings with GTK 2, so that's confusing. The import name is just gi, and the Debian packages are `python[3]-gi`. Pygi, with whatever capitalisation, doesn't seem to be in any way its name, but its the best way I can find to refer to it.